File: 6009.使两字符串互为字母异位词的最少步骤数/s.cpp
#include <iostream>
#include <string>
#include <vector>
#include <unordered_map>

using namespace std;

int main() { return 0; }

class Solution {
public:
    int minSteps(string s, string t) {
        vector<int> vs(26), vt(26);
        for (auto c : s) ++vs[c - 'a'];
        for (auto c : t) ++vt[c - 'a'];

        int res = 0;
        for (int i = 0; i < 26; i++) {
            res += abs(vs[i] - vt[i]);
        }
        return res;
    }
};

File: 6009.使两字符串互为字母异位词的最少步骤数/s.go
package main

func minSteps(s string, t string) int {
    ms, mt := [26]int{}, [26]int{}
    for _, c := range s {
		idx := int32(c) - int32('a')
        ms[idx]++
    }
    for _, c := range t {
        idx := int32(c) - int32('a')
        mt[idx]++
    }

    res := 0
    for i := 0; i < 26; i++ {
        res += Abs(ms[i], mt[i])
    }

    return res
}

func Abs(x, y int) int {
    if x > y {
        return x - y
    } 
    return y - x
}